### ISSUE TITLE
"cordova-plugin-file@6.0.2" does not satisfy dependency plugin requirement

### ISSUE THREAD

**Issue by hughknaus** [spoonconsulting/cordova-plugin-background-upload#236]
Trying to use `cordova-plugin-file-transfer` in the same project that I'm using `cordova-plugin-background-upload`.  However, with `ionic repair` the corodova prepare phase fails with:
```
Discovered plugin "cordova-plugin-file-transfer". Adding it to the project
Installing "cordova-plugin-file-transfer" for android
Failed to install 'cordova-plugin-file-transfer': CordovaError: Version of installed plugin: "cordova-plugin-file@6.0.2" does not satisfy dependency plugin requirement "cordova-plugin-file@>=7.0.0". Try --force to use installed plugin as dependency.
    at C:\Users\hknaus\AppData\Roaming\npm\node_modules\cordova\node_modules\cordova-lib\src\plugman\install.js:526:43
Failed to restore plugin "cordova-plugin-file-transfer". You might need to try adding it again. Error: CordovaError: Version of installed plugin: "cordova-plugin-file@6.0.2" does not satisfy dependency plugin requirement "cordova-plugin-file@>=7.0.0". Try --force to use installed plugin as dependency.
```
`cordova-plugin-background-upload` has a dependency on `cordova-plugin-file@6.0.2` while `cordova-plugin-file-transfer` has a dependency of `cordova-plugin-file@>=7.0.0`, how can these be used in the same project with the dependency disparity?

Snippet from my package.json:  
![image](https://user-images.githubusercontent.com/37626121/182954042-5d880802-63bc-4070-819b-118344935017.png)

**Comment by zafirskthelifehacker** [comment]
Hello @hughknaus,

You can try to increase the dependency on our plugin. (Create a PR and we will do some tests and merge it)

https://github.com/spoonconsulting/cordova-plugin-background-upload/blob/058291b70940cb74981befedd9b49afffb53b927/plugin.xml#L16

or you can do the reverse.

Regards,
Zafir

**Comment by hughknaus** [comment]
PR: Update dependency of cordova-plugin-file version -- https://github.com/spoonconsulting/cordova-plugin-background-upload/pull/237

**Comment by zafirskthelifehacker** [comment]
Hello @hughknaus,

Will test the PR and let you know.

Regards,
Zafir.

**Comment by zfir** [comment]
Hello @hughknaus,

This version of `cordova-plugin-file` has a problem when displaying an `img` in the WebView.

https://github.com/apache/cordova-plugin-file/issues/526

The link above address this issue.

Regards,
Zafir.

**Comment by hughknaus** [comment]
@zfir  Yeah, I believe we were running into it on Android as well.  iOS seemed to be okay though.

What are the plans for resolving these issues? Do you know the timeframe for resolution?

**Comment by zfir** [comment]
Hello @hughknaus,

No idea. We need to wait for Apache to release a new version. For now, I would recommend you to try to downgrade the dependency of `cordova-plugin-file-transfer` to `6.0.2` and test again.

Regards,
Zafir.

**Comment by gbrits** [comment]
I'm using `cordova-plugin-ns0m-file-transfer` - does this fix your problems? I am still running into a file not found issue though, but I'm thinking that's a URL `file://` thing that I can hopefully fix soon.

**Comment by MaximBelov** [comment]
fixed 
https://github.com/spoonconsulting/cordova-plugin-background-upload/pull/280
